src/app.js

```diff
@@ -23,6 +23,7 @@ function deobfuscate(e) {
         .replace(/Zero/g, "0")
 }
 $(function () {
+    var Waves
     Waves.displayEffect({ duration: 200 })
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') TypeError can be thrown as "Waves" might be null or undefined here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2259)